src/directory/file_slice.rs

```diff
@@ -82,7 +82,7 @@ impl FileSlice {
 
     /// Creates an empty FileSlice
     pub fn empty() -> FileSlice {
-        const EMPTY_SLICE: &'static [u8] = &[];
```

Comment:
That's not helping.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#redundant_static_lifetimes

Reply:
ok. still does not help, but sticking to clippy has values!

Reply:
OKAY, sounds too convincing to stand my ground